main.py

```diff
@@ -0,0 +1,50 @@
+from models.linked_list import ResistorLinkedList
+from models.resistor import Resistor
+from models.sort_order import SortOrder
+if __name__ == '__main__':
```

Comment:
There should be two blank lines after your imports

Reply:
Formatted the code